content/en/infrastructure/containers/configuration.md

```diff
@@ -76,6 +76,121 @@ password::::== <MY_PASSWORD>
 
 However, the scrubber does not scrub paths that contain sensitive words. For example, it does not overwrite `/etc/vaultd/secret/haproxy-crt.pem` with `/etc/vaultd/******/haproxy-crt.pem` even though `secret` is a sensitive word.
 
+## Configure Container Images View
+
+### Image Collection
+
+Datadog collects container image metadata to provide enhanced debugging context. 
+
+#### Agent
+
+{{< tabs >}}
+{{% tab "Kubernetes (Helm)" %}}
+
+If you are using helm version `>= 3.46.0`, image collection is [enabled by default][3].</br>
```

Comment:
```suggestion
If you are using Helm version `>= 3.46.0`, image collection is [enabled by default][3].</br>
```

Reply:
The link in this line is broken. Links inside tabs require you to put the link target inside the tab (before the closing tab shortcode).

Reply:
Just noticed that you did put the link inside the tab. Is it the same one? If so, update the link target to [1] instead of [3].